templates/iot/examples/iot-config-infinispan.yaml

```diff
@@ -19,3 +19,6 @@ spec:
       endpoint:
         secretNameStrategy:
           secretName: iot-mqtt-adapter-tls
+#  imageOverrides:
```

Comment:
I am not sure we should keep this in here. However I think it would make sense having a proper "override" example.

Reply:
I introduced `iot-config-dev.yaml` for easier dev deployments.